src/ucp/proto/proto_init.c

```diff
@@ -188,13 +188,12 @@ ucp_proto_perf_envelope_make(const ucp_proto_perf_list_t *perf_list,
 }
 
 ucs_status_t
-ucp_proto_init_parallel_stages(const ucp_proto_init_params_t *params,
-                               size_t range_start, size_t range_end,
+ucp_proto_init_parallel_stages(size_t range_start, size_t range_end,
```

Comment:
IMO it was better with single `params` argument, now it's already 8 args
It is also consistent with the rest of the functions taking the whole structure.
This way it's easier to introduce new changes without breaking the code
But up to you